Gini-iOS-SDK/GINIAPIManager.m

```diff
@@ -203,6 +203,32 @@ - (BFTask *)uploadDocumentWithData:(NSData *)documentData
     } cancellationToken:cancellationToken];
 }
 
+- (BFTask *)createCompositeDocumentWithPartialDocumentsInfo:(NSArray<NSDictionary<NSString *,id> *> *)partialDocumentsInfo
+                                                   fileName:(NSString *)fileName
+                                                    docType:(NSString *)docType
+                                          cancellationToken:(BFCancellationToken *)cancellationToken {   
+    NSDictionary* dict = @{@"partialDocuments": partialDocumentsInfo};
```

Comment:
This method is public (correct me if I'm wrong). In which case I find it dangerous to expect clients to pass in a `partialDocumentsInfo` which contains correct key names and values.

In my comment for the method declaration in the header file I suggested passing in an array of `PartialDocumentInfo` objects. Using something in the lines of that to separate the required json structure and the client supplied argument would avoid messing up the json structure. We would need to a little more overhead to serialize to json, but would be in control of it.